### ISSUE TITLE
White borders by default after applying notion-enhancer

### ISSUE THREAD

**Issue by naveenjohnsonv** [dragonwocky/notion-enhancer#95]
**describe the bug**
As soon as I apply notion-enhancer the app gets this weird borders which remain white irrespective of the theme. 
![image](https://user-images.githubusercontent.com/14140949/92587244-01b71700-f2b5-11ea-9660-d24c8109c25f.png)

**expected behaviour**
This is how notion looks like without notion-enhancer applied.
![image](https://user-images.githubusercontent.com/14140949/92587110-d16f7880-f2b4-11ea-9693-856a5ac9c61c.png)
Either those borders should not be present or they should also be themed according to the theme applied.

**platform**
Linux

**Comment by dragonwocky** [comment]
no idea where those borders are coming from, and can't replicate... are there any errors in the console? (`ctrl+shift+i`)
if you turn on smooth scrollbars in the enhancer menu (`ctrl+e`), does that change anything?

**Comment by naveenjohnsonv** [comment]
Refused to apply style from 'notion://www.notion.so/app-b1143cccdef6c905a0c7.css' because its MIME type ('') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
/opt/notion/app/renderer/preload.js:155 
Failed to load spellchecker Error: /opt/notion/app/node_modules/spellchecker/build/Release/spellchecker.node: invalid ELF header
    at process.func [as dlopen] (electron/js2c/asar.js:140)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1016)
    at Object.func [as .node] (electron/js2c/asar.js:140)
    at Module.load (internal/modules/cjs/loader.js:816)
    at Module._load (internal/modules/cjs/loader.js:728)
    at Function.Module._load (electron/js2c/asar.js:748)
    at Module.require (internal/modules/cjs/loader.js:853)
    at require (internal/modules/cjs/helpers.js:74)
    at Object.<anonymous> (/opt/notion/app/node_modules/spellchecker/lib/spellchecker.js:3)
    at Object.<anonymous> (/opt/notion/app/node_modules/spellchecker/lib/spellchecker.js:112)
loadSpellcheck @ /opt/notion/app/renderer/preload.js:155
app-97aad860c2e7cdb176ee.js:1 
2020-09-09T11:45:28.087Z [info] FeatureFlagTransactionQueue preferIndexedDB Object
security-warnings.ts:236 
Electron Security Warning (enableBlinkFeatures) This renderer process has additional "enableBlinkFeatures"
  enabled. This exposes users of this app to some security risk. If you do not
  need this feature, you should disable it.

For more information and help, consult
https://electronjs.org/docs/tutorial/security.
This warning will not show up
once the app is packaged.
warnAboutEnableBlinkFeatures @ security-warnings.ts:236
security-warnings.ts:181 Electron Security Warning (Insecure Content-Security-Policy) This renderer process has either no Content Security
    Policy set or a policy with "unsafe-eval" enabled. This exposes users of
    this app to unnecessary security risks.

For more information and help, consult
https://electronjs.org/docs/tutorial/security.
This warning will not show up
once the app is packaged.
(anonymous) @ security-warnings.ts:181
Journal-template-cb94298f427f4ab795009721c4887261:1 

Refused to apply style from 'notion://www.notion.so/app-b1143cccdef6c905a0c7.css' because its MIME type ('') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

DevTools failed to parse SourceMap: notion://www.notion.so/opt/notion/app/renderer/preload.js.map
DevTools failed to parse SourceMap: notion://www.notion.so/opt/notion/app/helpers/notionIpc.js.map
DevTools failed to parse SourceMap: notion://www.notion.so/opt/notion/app/node_modules/notion-intl/index.js.map
DevTools failed to parse SourceMap: notion://www.notion.so/opt/notion/app/helpers/localizationHelper.js.map
DevTools failed to parse SourceMap: notion://www.notion.so/opt/notion/app/shared/urlHelpers.js.map
vendors~main-4b715fb81452e0ba46a1.js:1 [Violation] 'setTimeout' handler took 62ms

**Comment by naveenjohnsonv** [comment]
Integrated scrollbars doesn't make a difference either. Are you on linux?

> no idea where those borders are coming from, and can't replicate... are there any errors in the console? (`ctrl+shift+i`)
> if you turn on smooth scrollbars in the enhancer menu (`ctrl+e`), does that change anything?

**Comment by dragonwocky** [comment]
i'm on windows, but in theory electron is meant to make things work as similarly as possible regardless of the platform. i know others on linux are able to run the enhancer fine...

**Comment by naveenjohnsonv** [comment]
> i'm on windows, but in theory electron is meant to make things work as similarly as possible regardless of the platform. i know others on linux are able to run the enhancer fine...

Is there any noticeable error on the console output?

**Comment by dragonwocky** [comment]
> Is there any noticeable error on the console output?
no - they're all normal errors for the linux build of the app (and most of them on windows too). sorry, should have said.

**Comment by naveenjohnsonv** [comment]
It's okay, how can I help find the problem now? Should I try reproducing this on another system?

**Comment by dragonwocky** [comment]
That'd be helpful.

**Comment by naveenjohnsonv** [comment]
Okay so I couldn't reproduce it on a usb boot of my same OS. So I figured it's definitely an issue on my side. So I manually removed Notion's cache, logged in again. The problem was fixed. Unfortunately, uninstalling apps on Linux doesn't remove it's cache. That's why I couldn't figure it out. Sorry for wasting your time.

![image](https://user-images.githubusercontent.com/14140949/92611506-80707c00-f2d6-11ea-9e3a-7bcaed0392a6.png)
